app/views/govuk_publishing_components/components/docs/accordion.yml

```diff
@@ -40,21 +40,21 @@ examples:
     data:
       items:
         - heading:
-            text: "Writing well for the web"
+            text: Writing well for the web
```

Comment:
Out of interest, why the all removal of the quotes? I thought we used them to avoid YAML's liberal use of things that could be booleans / Norway = false problem.

Reply:
This was based on recommendations by Andy that yml doesn't need quotes around single line text. If there's a standard that this goes against or we just think it's a bad idea I'm not precious about it.

Reply:
Andy's right, the quotes aren't needed in most cases - the main exception to this is that YAML tends to be quite enthusiastic about turning unquoted strings into booleans. There isn't a GDS Way standard, so I tend to follow the file's convention to avoid bike shedding.

Reply:
If you're ok with it I'm gonna leave this how it is. These strings don't seem to be incurring yml's wrath right now and it also fixes some general formatting in the doc itself. It's not a complex job to undo it if it's a problem and I'm not precious about this if you'd rather I change it now.

Reply:
Of course, that's absolutely fine - I was just wondering if there was some documentation somewhere or a discussion that I'd missed.